migrations/migrations/20210512201148-create-user.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      lastname: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true,
+        validate: {
+          isEmail: true
+        }
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      createdAt: {
```

Comment:
To respect the database conventions we decided to keep tables and columns in snake_case.

show 3.3- Database:
https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md